apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/SettingsActivity.java

```diff
@@ -137,6 +141,31 @@ public boolean isEnabled() {
                         }
                     }
                 });
+
+                final Method enableFirebaseAnalytics = push.getMethod("enableFirebaseAnalytics", Context.class);
+                initCheckBoxSetting(R.string.mobile_center_push_firebase_state_key, isFirebaseEnabled(), R.string.mobile_center_push_firebase_summary_enabled, R.string.mobile_center_push_firebase_summary_disabled, new HasEnabled() {
+
+                    @Override
+                    public void setEnabled(boolean enabled) {
```

Comment:
@bmourat this is what I had in mind when creating the task: https://github.com/bmourat/mobile-center-sdk-android/pull/2.

This will be better for our internal demos ;)